playbooks/common/openshift-node/config.yml

```diff
@@ -159,7 +159,6 @@
     openshift_nodes: "{{ hostvars
                          | oo_select_keys(groups['oo_nodes_to_config'])
                          | oo_collect('openshift.common.hostname') }}"
-    openshift_node_vars: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']) }}"
```

Comment:
:+1: on cleaning it up...  One thing I wonder is if we need to have some type of a filter to provide a more concise mapping of node -> label/scheduleable values. Right now we assume all of the node objects/vars are present, but maybe we should have a more strict definition of what we are acting on and enforce that at the playbook level.

This would also prevent transient node errors from potentially causing issues with the role as well.